lib/Image/Adapter.php

```diff
@@ -41,9 +41,6 @@ abstract class Adapter
 
     protected mixed $resource = null;
 
-    /**
-     * @return $this
-     */
```

Comment:
But they *have* an additional benefit, because `$this` is *not* the same as `static`. 

It says: this method returns exactly the same instance, not only an object of the same type.

So I'm voting for keeping them.

Reply:
> It says: this method returns exactly the same instance, not only an object of the same type.

It makes a difference when you have immutable objects: they return `static` (a new instance of the same type) while mutable objects can return `$this` (the same instance)

This is valuable info for static code analyzers like PHPStan.